phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/common/widget/MiniatureListWidget.java

```diff
@@ -61,7 +61,11 @@ private MiniatureListWidget(Pane content, Function<E, Element> converter, BiCons
         this.mappedElements = new MappedList<Element<E>, E>(items, value -> {
             Element newElement = converter.apply(value);
 
-            newElement.setOnMouseClicked(event -> setOnMouseClicked.accept(newElement, event));
+            newElement.setOnMouseClicked(event -> {
+                unselectAll();
+                setOnMouseClicked.accept(newElement, event);
+                select(newElement);
```

Comment:
Just as a note:
Is it possible that the select css class is the same as the hover one?
If this is true we should choose a different style for the selection css class, to make it more clear to the user I think.